instrumentation/opentelemetry-instrumentation-fastapi/src/opentelemetry/instrumentation/fastapi/__init__.py

```diff
@@ -226,9 +230,9 @@ def instrument_app(
         server_request_hook: ServerRequestHook = None,
         client_request_hook: ClientRequestHook = None,
         client_response_hook: ClientResponseHook = None,
-        tracer_provider=None,
-        meter_provider=None,
-        excluded_urls=None,
+        tracer_provider: TracerProvider | None = None,
```

Comment:
Nit: Could you use `typing.Optional` to be consistent with most other places in the code?

Reply:
Other places are out dated.

What should happen is to start switching the other places.

Reply:
Hard agreement

Reply:
For context the `from __future__ import annotations` let us use this syntax